docs/content/preview/yugabyte-voyager/migrate/migrate-steps.md

```diff
@@ -185,15 +189,109 @@ target:
   db-password: <target-db-password> # Enclose the password in single quotes if it contains special characters.
 ```
 
-Refer to the [offline-migration.yaml](https://github.com/yugabyte/yb-voyager/blob/{{< yb-voyager-release >}}/yb-voyager/config-templates/offline-migration.yaml) template for more information on the available global, source, and target configuration parameters supported by Voyager.
+Refer to the [offline-migration.yaml](<https://github.com/yugabyte/yb-voyager/blob/{{>< yb-voyager-release >}}/yb-voyager/config-templates/offline-migration.yaml) template for more information on the available global, source, and target configuration parameters supported by Voyager.
 
 ## Assess migration
 
-This step is optional and only applies to PostgreSQL and Oracle migrations.
+This step applies to PostgreSQL and Oracle migrations only.
+
+Assess migration analyzes the source database, captures essential metadata, and generates a report with recommended migration strategies and cluster configurations for optimal performance with YugabyteDB.
+
+### Configure yugabyted UI
```

Comment:
Why are we repeating all this here rather than xref to assess page where it is already described